### ISSUE TITLE
[Question]How to sync files on the same machine hosting Nextcloud with client?

### ISSUE THREAD

**Issue by yunlhan** [nextcloud/client_theming#218]
Hello,

I recently upgraded Nextcloud server to 12.0.3 (manual upgrade, everything went fine; server runs on Debian stretch), but when I tried to use `Nextcloud-2.3.3-beta-x86_64.AppImage` (downloaded from https://download.nextcloud.com/desktop/prereleases/Linux/) to sync files **from the server on the same machine as the server**, I got the error

`timeout trying to connect to nextcloud at https://cloud.MYDOMAIN.com/status.php`

However if use a web browser and go to 

`https://cloud.MYDOMAIN.com/status.php`

I can get something from `status.php`:

`
installed | true
maintenance | false
needsDbUpgrade | false
version | "12.0.3.3"
versionstring | "12.0.3"
edition | ""
productname | "Proudly powered by Debian"
`

So basically I want to work on a machine itself that hosts `nextcloud`. Can anyone point me to the solution to this problem?

Thanks,
Yün

**Comment by yunlhan** [comment]
Figured it out! It was not a problem of nextcloud. The server generally does not resolve self IP if external DNS is used. Added `localhost` to `server_name` directive in nextcloud nginx config file; it fixed the problem. Thanks.
